modules/spo-inspecting-seccomp-profiles.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="spo-inspecting-seccomp-profiles_{context}"]
+= Inspecting seccomp profiles
+
+Corrupted `seccomp` profiles can disrupt your workloads. Ensure that the user cannot abuse the system by not allowing other workloads to map any part of the path `/var/lib/kubelet/seccomp/operator`.
+
+.Procedure
+
+. Confirm that the profile is reconciled by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-security-profiles logs openshift-security-profiles-mzw9t
```

Comment:
Can the `-mzw9t` part be genericized? E.g., `openshift-security-profiles-<id>`

Same for `-v6p2h` later.